cypress/plugins/docker/index.ts

```diff
@@ -36,12 +36,16 @@ export async function dockerRun(opts: {
     const params = opts.params ?? [];
 
     if (params?.includes("-v") && userInfo.uid >= 0) {
-        // On *nix we run the docker container as our uid:gid otherwise cleaning it up its media_store can be difficult
-        params.push("-u", `${userInfo.uid}:${userInfo.gid}`);
-
         if (await isPodman()) {
-            // keep the user ID if the docker command is actually podman
-            params.push("--userns=keep-id");
+            // In podman, run as root in the container, so we're the current
+            // user on the host
+            params.push("-u", "0:0");
```

Comment:
is this not the default? https://docs.podman.io/en/latest/markdown/podman-run.1.html#user-u-user-group seems to say so.

(I don't mind if you want to make it explicit, but please comment that is what you are doing)

Reply:
I'm not sure. I don't read that page as "root is the default".

Reply:
> Without this argument, the command will run as the user specified in the container image. Unless overridden by a USER command in the Containerfile or by a value passed to this option, this user generally defaults to root.

Synapse doesn't set `USER`, so it sounds like it defaults to root to me?

(Though I wish the 'generally' was a bit more confident)

Reply:
Ah, I see what you mean. I'll add a comment.